lib/matplotlib/colors.py

```diff
@@ -1213,7 +1213,7 @@ class SymLogNorm(Normalize):
     *linthresh* allows the user to specify the size of this range
     (-*linthresh*, *linthresh*).
     """
-    def __init__(self, linthresh, linscale=1.0,
+    def __init__(self, linthresh, linscale=1.0, base=None,
```

Comment:
This needs to go at the end.

Reply:
Done...

Reply:
Is there a guide on which arguments we may/want to make keyword-only?

Reply:
Not really, but in this case, we can't really make any of then kwarg only because of back-compatibility concerns.

Reply:
I am talking only about the added one.

Reply:
As proposed in #16181, I‘d make almost all arguments kw-only. This will in particular also hold for added parameters.

Reply:
Sure, are you guys asking that just the new param is kwarg only, or all of them?

Reply:
For now, only the new parameter base can be kw-only. Making other parameters kw-only needs a deprecation first.